Arrays/3443MaximumManhattanDistanceAfterKChanges.py

```
@@ -0,0 +1,106 @@
+                  if temp_resultNS[0][0]==max_distance:
+                      s[temp]='N' if s[temp]=='S' else 'S'
+                      k+=1
+                  elif temp_resultNS[0][0]>max_distance:
+                      max_distance=temp_resultNS[0][0]
+                      resultNS=temp_resultNS
+             else:
+                 break
+             print(max_distance) 
+        print(max_distance)
+        return max_distance 
+Solution().maxDistance("NWSE",1)
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 0

(E305)